L1Trigger/L1THGCal/test/testHGCalL1TGeometryV9_cfg.py

```diff
@@ -0,0 +1,122 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process('DIGI',eras.Phase2)
```

Comment:
I think this was changed by mistake. `Phase2C4` is the correct era for HGCal v9.

Reply:
Fixed